docs/wsl2/wsl2.md

```diff
@@ -0,0 +1,186 @@
+# Using `cog` on Windows 11 with WSL 2
+
+- [1. Install the GPU driver](#1-install-the-gpu-driver)
+- [2. Unlocking features](#2-unlocking-features)
+- [3. Update MS Linux kernel](#3-update-ms-linux-kernel)
+- [4. Configure WSL 2](#4-configure-wsl-2)
+- [5. Configure CUDA WSL-Ubuntu Toolkit](#5-configure-cuda-wsl-ubuntu-toolkit)
+- [6. Install Docker](#6-install-docker)
+- [7. Install `cog` and pull an image](#7-install-cog-and-pull-an-image)
+- [8. Run a model in WSL 2](#8-run-a-model-in-wsl-2)
+- [9. References](#9-references)
+
+Running cog on Windows is now possible thanks to WSL 2. Follow this guide to enable WSL 2 and GPU passthrough on Windows 11. 
+
+Windows 10 is not officially supported, as you need to be on an insider build in order to use GPU passthrough.
```

Comment:
Maybe not necessary, but do you think it's worth having a section for prerequisites at the start here, so people know what they'll need before they get too deep?

Reply:
Yeah, definitely. I added a section to the top explaining the requirement for Win11 and that you need an NVIDIA GPU.